addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -159,12 +159,15 @@ def attributes(self, record, field_name, options, values):
 
         if options['translate'] and field.type in ('char', 'text'):
             lang = record.env.lang or 'en_US'
-            if lang == 'en_US':
+            lang_base = 'en_US'
+            if 'website_id' in record._fields:
+                lang_base = record.website_id.default_lang_id.code
```

Comment:
@odoo/rd-website 
I don't know if this condition is enough to handle most uses cases when we want to get the record's bounded website's default language.
Do you have any suggestion?

Reply:
I'm not from the website team but note that `web_editor` does not depend on `website`.

Reply:
You can't find the website like that as most records don't even have a `website_id`, it will most of the time do nothing.

You might want to use the `ir_http.get_request_website()` util to retrieve it from the request (it takes care of ensuring the request exists).

Out of curiosity, don't you need to flag a record as translated or not somewhere in the backend? How do you do that if the database is started in a language other than english?

Reply:
> Out of curiosity, don't you need to flag a record as translated or not somewhere in the backend? How do you do that if the database is started in a language other than english?

We don't save the flag because
1. it will make the jsonb more complicated than the current `{"en_US": "English", "fr_FR": "French"}`
2. since there is no `ir.translation` model for all translations, users can to use a filter get all untranslated terms. The 'to_translate' tag is only used by website in edit_translation mode